config/ember-try.js

```diff
@@ -10,62 +10,17 @@ module.exports = function() {
     return {
       useYarn: true,
       scenarios: [
-        {
-          name: 'ember-lts-2.18',
-          npm: {
-            devDependencies: {
-              '@ember/jquery': '^1.1.0',
-              '@ember/test-helpers': '^1.7.0',
-              'ember-a11y-testing': '^0.5.0',
-              'ember-angle-bracket-invocation-polyfill': '^3.0.1',
-              'ember-qunit': '^4.0.0',
-              'ember-source': '~2.18.0',
-              qunit: null,
-            },
-            resolutions: {
-              'ember-classy-page-object': '0.7.0',
-            },
-          },
-        },
-        {
-          name: 'ember-lts-3.4',
-          npm: {
-            devDependencies: {
-              'ember-source': '~3.4.0',
-            },
-            dependencies: {
-              'ember-classy-page-object': '0.7.0',
-            },
-            resolutions: {
-              'ember-classy-page-object': '0.7.0',
-            },
-          },
-        },
-        {
-          name: 'ember-lts-3.8',
-          npm: {
-            devDependencies: {
-              'ember-source': '~3.8.0',
-            },
-            dependencies: {
-              'ember-classy-page-object': '0.7.0',
-            },
-            resolutions: {
-              'ember-classy-page-object': '0.7.0',
-            },
-          },
-        },
         {
           name: 'ember-lts-3.12',
           npm: {
             devDependencies: {
               'ember-source': '~3.12.0',
             },
             dependencies: {
-              'ember-classy-page-object': '0.7.0',
+              'ember-classy-page-object': '0.8.0-beta.2',
```

Comment:
If these all match the default version found in `package.json` now, then we shouldn't need to override them here, right?

Reply:
https://github.com/Addepar/ember-table#using-ember-table-with-ember--324 is probably still relevant, though obviously the test suite is green. Can you rationalize this @twokul? I think we were limited to `0.7.0` on Ember < 3.24 based on upstream support in ember-cli-page-object.

Reply:
Yeah, you are 100% correct. I think we can just "remove and forget." Not sure how I missed this.